startup.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-python3 -m app.consumer
\ No newline at end of file
+python3 -m app.main
```

Comment:
This will break the deployment build in Jenkins (it looks for a consumer module) - so you may need to look at that once this is merged in. I would leave this as main though, it makes more sense